charts/clearblade-iot-enterprise/charts/cb-postgres/templates/postgres-cluster.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if .Values.global.dynamicVolumes }}
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cb-postgres
+  namespace: {{ default "clearblade" .Values.global.namespace }}
+  labels:
+    app: cb-postgres
+  annotations:
+    cnpg.io/skipEmptyWalArchiveCheck: enabled
+spec:
+  # template:
+  #   spec:
+  #     serviceAccountName: {{ .Values.global.gcpGSMServiceAccount }}
+        # tolerations:
+        # - key: node.kubernetes.io/not-ready
+        #   operator: Exists
+        #   effect: NoExecute
+        #   tolerationSeconds: 0
+        # - key: node.kubernetes.io/unreachable
+        #   operator: Exists
+        #   effect: NoExecute
+        #   tolerationSeconds: 0
```

Comment:
Don't we want this commented out code to prevent the issues with pods getting stuck as not ready?